src/email.rs

```diff
@@ -93,7 +90,7 @@ pub fn request(app: &Config, email_addr: EmailAddress, client_id: &str, nonce: &
 pub fn verify(app: &Config, stored: &HashMap<String, String>, code: &str)
               -> BrokerResult<(String, String)> {
 
-    if code != &stored["code"] {
+    if code.to_lowercase() != (&stored["code"]).to_string() {
```

Comment:
I agree that it'd clutter the code, for fairly small breakage during beta. I'd be more worried if we were 1.0. :)